hazelcast/src/test/java/com/hazelcast/replicatedmap/ReplicatedMapMergePolicyTest.java

```diff
@@ -77,30 +69,38 @@ public void killAllHazelcastInstances() {
     @Parameterized.Parameter
     public ReplicatedMapMergePolicyTestCase testCase;
 
+    private TestHazelcastInstanceFactory factory;
```

Comment:
Termination is being handled by `HazelcastTestSupport.shutdownNodeFactory()` automatically. `HazelcastTestSupport.createHazelcastInstanceFactory()` also sets the created factory class into the internal field so that it will be able to terminate it automatically.